### ISSUE TITLE
What are the differences between `mysql` & `postgres` renderer?

### ISSUE THREAD

**Issue by andinus** [Leont/sql-abstract#1]
Before this commit: https://github.com/Leont/sql-abstract/commit/3e50f5e71adad9cec19981ee03645b8798bd154a

We created SQL::Abstract object like so:
```
        $!abstract = SQL::Abstract.new(:placeholders<dbi>);
```

Adding `mysql` as the renderer breaks some code but keeping the previous default (`:renderer<postgres>`) works.

This is the code:
```
.update(
    %( :$client_secret, :$intent_id, ),
    %( :user_id($session.id), :$id ),
);
```

Here is the error when we use `mysql` as the renderer:
```
Cannot resolve caller render-source(SQL::Abstract::Renderer::MySQL:D: SQL::Abstract::Table::Simple:D); none of these signatures matches:
(SQL::Abstract::Renderer::MySQL: SQL::Placeholders, SQL::Abstract::Table $table, *%_ --> List)
(SQL::Abstract::Renderer::MySQL: SQL::Placeholders $placeholders, SQL::Abstract::Source::Nested $nested, *%_ --> List)
(SQL::Abstract::Renderer::MySQL: SQL::Placeholders $placeholders, SQL::Abstract::Join $join, *%_ --> List)
in method render-expression at /home/raku/.raku/sources/F67F30596889081A04173E97B431D0A435C6C048 (SQL::Abstract) line 2224
in method render-expression at /home/raku/.raku/sources/F67F30596889081A04173E97B431D0A435C6C048 (SQL::Abstract) line 1717
in method render at /home/raku/.raku/sources/F67F30596889081A04173E97B431D0A435C6C048 (SQL::Abstract) line 2128
in method update at /home/raku/.raku/sources/F67F30596889081A04173E97B431D0A435C6C048 (SQL::Abstract) line 2299
[...]
```

Is the update code supposed to be different when using `mysql` renderer?

**Comment by Leont** [comment]
That was just a silly mistake on my side, this should be fixed in 0.009 by 8ef687e76fab0ed177fb2c369386366a2048d8ee

**Comment by andinus** [comment]
Thank you!
